extensions/iceberg/s3/src/test/java/io/deephaven/iceberg/util/IcebergToolsTest.java

```diff
@@ -126,13 +126,14 @@ void setUp() throws Exception {
 
         final S3Instructions s3Instructions = s3Instructions(S3Instructions.builder()).build();
 
-        instructions = IcebergInstructions.builder()
+        instructions = IcebergReadInstructions.builder()
                 .dataInstructions(s3Instructions)
                 .build();
     }
 
     @AfterEach
-    void tearDown() throws ExecutionException, InterruptedException {
+    void tearDown() throws Exception {
+        engineCleanup.tearDown();
```

Comment:
Typically, tearDown should happen in the reverse order that setup happens; ie, `engineCleanup` should happen last.